README.md

```diff
@@ -354,7 +354,7 @@ There're also some great commercial libraries, like [amchart](https://www.amchar
 * [lodash](https://github.com/lodash/lodash) - A utility library delivering consistency, customization, performance, & extras.
 * [Sugar](https://github.com/andrewplummer/Sugar) - A Javascript library for working with native objects.
 * [lazy.js](https://github.com/dtao/lazy.js) - Like Underscore, but lazier.
-* [ramda](https://github.com/CrossEye/ramda) - A practical functional library for Javascript programmers.
+* [ramda](https://github.com/ramda/ramda) - A practical functional library for Javascript programmers.
```

Comment:
> JavaScript

Can you update it like this? Thanks!

Reply:
I made another PR that updates all of them. https://github.com/sorrycc/awesome-javascript/pull/598